staging/src/k8s.io/apiserver/pkg/server/options/serving_with_loopback.go

```diff
@@ -35,16 +36,16 @@ func WithLoopback(o *SecureServingOptions) *SecureServingOptionsWithLoopback {
 }
 
 // ApplyTo fills up serving information in the server configuration.
-func (s *SecureServingOptionsWithLoopback) ApplyTo(c *server.Config) error {
-	if s == nil || s.SecureServingOptions == nil {
+func (s *SecureServingOptionsWithLoopback) ApplyTo(secureServingInfo **server.SecureServingInfo, loopbackClientConfig **rest.Config) error {
```

Comment:
Can we create a new method for this?  I accept that it's not specific to loopback, but the symmetric `ApplyTo` on all options is very easy to follow.  Then this could be `ApplyToServingInfo` or some such.  Still available, but the symmetric calls stay symmetric.

Reply:
I would prefer to leave this as it is in this PR. We have more of those cases. We should uniformely change the names. Also this would have ripples into a number of other components.

Reply:
> I would prefer to leave this as it is in this PR. We have more of those cases. We should uniformely change the names. Also this would have ripples into a number of other components.

Ok.